packages/rest/test/integration/rest.server.integration.ts

```diff
@@ -259,6 +259,7 @@ servers:
     const server = await givenAServer({
       rest: {
         port: 0,
+        host: process.env.TRAVIS ? '127.0.0.1' : undefined,
```

Comment:
Similarly here - don't duplicate this check, create a helper method to create http-server config.

I think it would be best for `@loopback/testlab` to implement the function `givenServerConfig` mentioned in my earlier comment - that way we can use it in any test.